integreat_compass/cms/urls/protected.py

```diff
@@ -0,0 +1,4 @@
+"""
+URLconf for login-protected views of the cms package.
+"""
+urlpatterns = []
```

Comment:
```suggestion

# TODO: Add middleware to actually enforce access control on these views!

urlpatterns = []
```
I'd point out that these views are not proteced (yet)...

Reply:
Haha, fair point... 😅